s3/s3.go

```diff
@@ -1 +1,100 @@
 package s3
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/credentials"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/s3"
+	"github.com/opensaucerer/bifrost/shared/errors"
+	"github.com/opensaucerer/bifrost/shared/types"
+)
+
+func (s SimpleStorageService) UploadFile(path, filename string) (types.UploadedFile, error) {
```

Comment:
Let's make the receiver a pointer `*SimpleStorageService` for flexibility in case we want to provide a method some time in the future to allow users to change preset configuration on the fly.